src/lib/storage/chunk.hpp

```diff
@@ -13,37 +13,37 @@
 #include <string>
 #include <vector>
 
-#include "index/column_index_type.hpp"
+#include "index/segment_index_type.hpp"
 
 #include "all_type_variant.hpp"
 #include "chunk_access_counter.hpp"
-#include "mvcc_columns.hpp"
-#include "table_column_definition.hpp"
+#include "mvcc_data.hpp"
+#include "table_cxlumn_definition.hpp"
 #include "types.hpp"
 #include "utils/copyable_atomic.hpp"
 #include "utils/scoped_locking_ptr.hpp"
 
 namespace opossum {
 
 class BaseIndex;
-class BaseColumn;
+class BaseSegment;
 class ChunkStatistics;
 
-using ChunkColumns = pmr_vector<std::shared_ptr<BaseColumn>>;
+using Segments = pmr_vector<std::shared_ptr<BaseSegment>>;
```

Comment:
@mrks is this a good idea? Maybe "ChunkSegments" or "Chunk::Segments"?

Reply:
I would stay with `Segments  `as the concept of `Segments` is only used within chunks.

Reply:
mmpf

Reply:
Same. I actually think that we should not use the term "segment" outside of its new definition. The only place where this is the case is `UnionPositions`. What is the state of that? Do we still need it? If yes, I think we should rename the `ColumnSegments` there.